miden-lib/asm/miden/account.masm

```diff
@@ -101,6 +101,37 @@ export.set_item
     # => [R', V]
 end
 
+#! Gets a map item from the account storage. Panics if the index is out of bounds.
+#!
+#! Stack: [index, KEY]
+#! Output: [VALUE]
+#!
+#! - index is the index of the item to get.
+#! - VALUE is the value of the item.
+export.get_map_item
+    push.0 movdn.5 push.0 movdn.5 push.0 movdn.5
+    # => [index, KEY, 0, 0, 0]
+    syscall.get_account_map_item
+    # => [VALUE]
```

Comment:
**Request**: If we do change the output of `get_account_map_item`, this will need to clear the stack, with something like:

```
movup.5 drop
```

Reply:
Hmm let's have a call about those change. They don't seem to work or I don't understand them correctly.

Reply:
done